Assets/Scripts/Waypoint Scripts/UpdateWaypoints.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+namespace ISAACS
+{
+    public class UpdateWaypoints : MonoBehaviour
```

Comment:
What is an UpdateWaypoints? Seems more like a method name than a class name

Reply:
@apollovision

Reply:
It's what add the lines between waypoints, but we need to seriously clean it up and document it.

Reply:
Agreed, I'm unsure of whether it should be separate from Waypoint.cs

Reply:
It seems to imply it's attached to the drone prefab but I don't see it being used anywhere.

Reply:
I deleted the script and everything still works, we should check once in VR and delete if it's unnecessary.

Reply:
We will work on the waypoint scripts in a new pull request.